README.md

```diff
@@ -271,6 +271,8 @@ Use one of Datadog's integrations to automatically track views for the following
 - If you use the [`react-native-navigation`][5] library, then add the `@datadog/mobile-react-native-navigation` package and follow the [setup instructions][6].
 - If you use the [`react-navigation`][7] library, then add the `@datadog/mobile-react-navigation` package and follow the [setup instructions][8].
 
+Refer to the [`react navigation example apps`](https://github.com/DataDog/dd-sdk-reactnative-examples) to see an implementation.
```

Comment:
Maybe the wording is a bit strong here. The example apps are going to be very helpful if something is wrong in the user setup, but by default reading the docs should be enough, we should not imply that they **have to** see the example repo unless they experience issues.

Maybe we use something like this

```suggestion
If you experience any issue setting up View tracking with `@datadog/mobile-react-navigation` you can check out our [example application](https://github.com/DataDog/dd-sdk-reactnative-examples/tree/main/rum-react-navigation) 
```

This comment applies to all changes

Reply:
In your wording use the official package name or library name for external libraries, i.e. `react-navigation` or `React Navigation` instead of `react navigation`.

Reply:
For links in docs we use a specific pattern. 
It should look like this:

```markdown
Here is a [link to some content][1]

<!-- end of document -->
[1]: https://some.content
```

I think this needs to be addressed for other links as well :)